npm-shrinkwrap.json

```diff
@@ -1363,7 +1573,7 @@
       "version": "1.1.7"
     },
     "graceful-fs": {
-      "version": "4.1.4"
+      "version": "1.2.3"
```

Comment:
ah, yeah, node-sass is the main culprit. looks like that will get bumped when we move to node 6 in #5595 so yay